lib/plugins.js

```diff
@@ -467,6 +475,38 @@ function plugins (settings, deviceId) {
     })
   }
 
+  function truncateCrypto (cryptoAtoms, cryptoCode) {
+    const DECIMAL_PLACES = 3
+    if (cryptoAtoms.eq(0)) return cryptoAtoms
+
+    const scale = 5 // TODO: change this to coins.displayScale when coins have that attribute
```

Comment:
What about this TODO here?

Reply:
displayScale on coins is only available on the machine side, so the server needs to know about its value somehow. This is solved on the l-coins module, which is not available here. What would be the optimal way in the meantime?